manifests/ocs/cluster.yaml

```diff
@@ -0,0 +1,46 @@
+apiVersion: ceph.rook.io/v1
+kind: CephCluster
+metadata:
+  name: rook-ceph
```

Comment:
Ceph cluster name and namespace should match ( see https://github.com/rook/rook/issues/3344 ) and in this case, 'openshift-storage' is required for the namespace, so the ceph cluster name should be 'openshift-storage' as well.